### ISSUE TITLE
use:hotkey not working

### ISSUE THREAD

**Issue by LukasDoesDev** [svelteuidev/svelteui#116]
### What package has an issue

@svelteuidev/core

### A clear and concise description of what the bug is

The code https://github.com/svelteuidev/svelteui/blob/dfad983a368c2ff40a54804a495f8c9a987b5c37/packages/svelteui-demos/src/lib/demos/core/AppShell/_HeadContent.svelte#L31= doesn't work for me
I'm getting the error
```ts
Type '(node: HTMLElement, hotkeys: HotkeyItem[]) => { update?: ((parameters: any) => any) | undefined; destroy?: (() => void) | undefined; }' is not assignable to type 'SvelteActionType<any>'.
```

### Steps To Reproduce

1. Copy the code at https://github.com/svelteuidev/svelteui/blob/dfad983a368c2ff40a54804a495f8c9a987b5c37/packages/svelteui-demos/src/lib/demos/core/AppShell/_HeadContent.svelte
2. See the compiler error

### Do you know how to fix the issue

No

### Are you willing to participate in fixing this issue and create a pull request with the fix

Yes

**Comment by Brisklemonade** [comment]
It's not an actual error, just a typescript error. So, that means the fix lies in modifying the types here:

https://github.com/svelteuidev/svelteui/blob/main/packages/svelteui-core/src/lib/internal/utils/forwarding/forward-actions.ts

**Comment by LukasDoesDev** [comment]
Seems like there is a mismatch between
https://github.com/svelteuidev/svelteui/blob/3869ed04ab73cb04d057c6913a4eff8dbf762b8a/packages/svelteui-composables/src/lib/shared/actions/types/ActionTypes.d.ts#L1-L7
and
https://github.com/svelteuidev/svelteui/blob/3869ed04ab73cb04d057c6913a4eff8dbf762b8a/packages/svelteui-core/src/lib/internal/utils/forwarding/forward-actions.ts#L5-L8

**Comment by JasonXu314** [comment]
I actually think its an issue with 
https://github.com/svelteuidev/svelteui/blob/3869ed04ab73cb04d057c6913a4eff8dbf762b8a/packages/svelteui-core/src/lib/internal/utils/forwarding/forward-actions.ts#L10

As far as I can tell, all of the actions are defined as functions taking a `HTMLElement` as the first parameter, which conflicts with the defined parameter of `HTMLOrSVGElement`. Svelte's definition of `Actions` actually has them taking in a `Node` as their first argument, so perhaps we could change the `Action` type definition to use that instead?

**Comment by Brisklemonade** [comment]
@JasonXu314 good observation, would you like to submit a PR for this fix?

**Comment by Brisklemonade** [comment]
Yeah, the typings I think are different between the core package and composables package. If you can change both action definitions to be unified and send in a PR, that would be AMAZING. @JasonXu314
